crates/bevy_pbr/src/render/mesh/mod.rs

```diff
@@ -276,12 +260,20 @@ pub fn extract_skinned_meshes(
     commands.insert_or_spawn_batch(values);
 }
 
+/// All possible [`BindGroupLayout`]s in bevy's default mesh shader (`mesh.wgsl`).
+#[derive(Clone)]
+pub struct MeshLayouts {
```

Comment:
I would rename this to MeshBindGroupLayouts. Then, I feel like this should be moved to `bindings.rs`, and the current nested modules replaced with associated static functions on this type.

Reply:
Making the `layout` (and `group`) associated functions of `MeshLayouts` is a great idea, it makes the code more rusty, and also it's easier to use without accidentally mixing up bind group layouts.

I prefer `MeshLayouts` over `MeshBindGroupLayouts` though, I find it hard to visually distinguish from `BindGroupLayout` otherwise. The `BindGroupLayout` overwhelms the `Mesh` bit and it becomes difficult to notice that `MeshBindGroupLayouts` is different from `BindGroupLayouts`, especially in struct declarations and function parameters where they are near each other and repeated. It's the "letter soup syndrome".